examples/ConwaysGameOfLife/Readme.md

```diff
@@ -0,0 +1,15 @@
+## Conway's Game Of Life
+
+[The Game of Life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life), also known simply as Life, is a cellular automaton devised by the British mathematician John Horton Conway in 1970.
+
+The "game" is a zero-player game, meaning that its evolution is determined by its initial state, requiring no further input. One interacts with the Game of Life by creating an initial configuration and observing how it evolves, or, for advanced "players", by creating patterns with particular properties.
```

Comment:
maybe... 'requiring no further input by a human'. <--- I feel like it needs a little more clarification. Yes? No?